addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -64,9 +64,9 @@
             </t>
             <div t-if="props.data.tax_details.length > 0" class="pos-receipt-taxes">
                 <span />
-                <span>VAT%</span>
-                <span>VAT</span>
-                <span>ExVAT</span>
+                <span>Tax%</span>
```

Comment:
Just wait need to check if this works well on POS
POS might have a spacing issue might need to replace it with CAPS

Reply:
![image](https://github.com/odoo/odoo/assets/121081873/d25aa287-7841-4594-9d74-745bc9596511)
Seems like it is a CSS spacing issue, changing header to caps should work as of now :face_exhaling: